ecs-init/engine/engine.go

```diff
@@ -33,6 +35,11 @@ const (
 	terminalSuccessAgentExitCode = 0
 	terminalFailureAgentExitCode = 5
 	upgradeAgentExitCode         = 42
+	minServiceStartRetryTime     = time.Millisecond * 500
+	maxServiceStartRetryTime     = time.Second * 15
+	serviceStartRetryJitter      = 0.10
+	serviceStartRetryMultiplier  = 2.0
+	serviceStartMaxRetries       = 999999999
```

Comment:
This const *really* should be configurable, if nothing else then it should be `math.MaxInt64` to convey that we're not choosing an arbitrarily large number - its intended to be "forever". Perhaps add a comment regarding this decision?

Reply:
yes I can use math.MaxInt64